wildfly/unifiedpush-wildfly/Dockerfile

```diff
@@ -2,14 +2,32 @@
 FROM aerogear/wildfly
 MAINTAINER Bruno Oliveira <bruno@aerogear.org>
 
+
+# Download Aerogear distribution
+ENV UPSVER=1.1.1.Final
```

Comment:
this naming schema _should_ be the correct one.

Yes, there were a few diffs, but should not happy anymore :)